pytest_selenium/driver.py

```diff
@@ -44,9 +47,13 @@ def make_driver(item):
 
 def start_driver(item, capabilities):
     options = item.config.option
-    specific_driver = '%s_driver' % options.driver.lower()
-    driver_method = globals().get(specific_driver, generic_driver)
-    driver = driver_method(item, capabilities)
+    driver_name = (options.driver and options.driver.lower()
+                   or None)
+    if driver_name not in [browser.lower() for browser in SUPPORTED_DRIVERS]:
```

Comment:
I'd prefer `if options.driver and options.driver.lower() not in [d for d in SUPPORTED_DRIVERS]`